core/util/inAppLink.jsx

```diff
@@ -1,9 +1,40 @@
+function openUrl(url) {
+  SafariViewController.isAvailable(function (available) {
```

Comment:
err. well. it is wrapper in a `Meteor.isCordova`, so maybe you were thinking in both? @jbaxleyiii